miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/QcTypeDefaultsController.java

```diff
@@ -0,0 +1,69 @@
+package uk.ac.bbsrc.tgac.miso.webapp.controller;
+
+import java.io.IOException;
+import java.util.Collection;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import uk.ac.bbsrc.tgac.miso.core.data.type.QcType;
+import uk.ac.bbsrc.tgac.miso.dto.Dtos;
+import uk.ac.bbsrc.tgac.miso.dto.QcTypeDto;
+import uk.ac.bbsrc.tgac.miso.service.QcTypeService;
+import uk.ac.bbsrc.tgac.miso.service.QualityControlService;
+
+@Controller
+@RequestMapping("/qctype")
+public class QcTypeDefaultsController extends AbstractInstituteDefaultsController<QcType, QcTypeDto> {
+  @Autowired
+  private QcTypeService qcTypeService;
+
+  @Autowired
+  private QualityControlService qcService;
+
+  @Override
+  protected QcTypeDto asDto(QcType model) {
+    return Dtos.asDto(model);
+  }
+
+  @Override
+  protected QcType get(long id) throws IOException {
+    return qcTypeService.get(id);
+  }
+
+  @Override
+  protected Collection<QcType> getAll() throws IOException {
+    return qcTypeService.getAll();
+  }
+
+  @Override
+  protected QcTypeDto getBlankModel() {
+    return new QcTypeDto();
+  }
+
+  @Override
+  protected Class<QcTypeDto> getDtoClass() {
+    return QcTypeDto.class;
+  }
+
+  @Override
+  protected String getName() {
+    return "Qc Type";
+  }
+
+  @Override
+  protected String getType() {
+    return "qctype";
+  }
+
+  @Override
+  protected void writeConfiguration(ObjectMapper mapper, ObjectNode config) throws IOException {
+    super.writeConfiguration(mapper, config);
+    config.putPOJO("qcTargets", qcService.listQcTargets());
```

Comment:
I'm not sure there's a compelling reason to have a `listQcTargets` method rather than just using `QcTarget.values()` here.

Also, since there aren't going to change, it's probably more sensible to put them in the `Constants` structure, set up in `MenuController`.